frontend/README.md

```diff
@@ -40,3 +40,63 @@ yarn clean # clear previous build artifacts
 2. Fill in the `.env` file with your Infura ID. You only need the Portis and Fortmatic API keys if you plan on using those wallets
 3. Install dependencies with `yarn`
 4. Build for development with `yarn run dev`
+
+## Internationalization
+
+The app is currently available in English and Simplified Chinese.
+
+Basic usauge is as follows:
```

Comment:
```suggestion
Basic usage is as follows:
```

Reply:
Also can you add a "For more details on the internationalization approach see the <package> docs and <these other> docs" with links to the documentation for whichever packages we have?

Reply:
Added here https://github.com/ScopeLift/umbra-protocol/pull/331/commits/9f0d839d6c20685fdfec9d4fc75085d012a2c795